docs/howto/openstack/nova/new-server.md

```diff
@@ -0,0 +1,337 @@
+	its public IP  address (e.g., from the _Addresses_ tab) and connect to 
+	it remotely. Alternatively, you may launch a web console and log in; 
+	click on the three-dot icon on the right of the server header, and from 
+	the pop-up menu that appears select _Remote Console_.  
+		
+	![Launch remote console](assets/new-server/shot-10.png)
+		
+	A new window pops up, and that's your web console to your 
+	{{extra.brand}} Cloud server. Please note that this window cannot be 
+	resized but can be opened on a new browser window or tab.
+		
+	![Server console](assets/new-server/shot-11.png)
+		
+	At any time ---and from the comfort of the {{extra.brand}} 
+	Cloud Control Panel--- you may start, stop, pause, reboot,
+	or delete any of your servers.
+=== "OpenStack CLI"
+	An `openstack` command for creating a server, may look like 
+	this:
+		
+	```bash
+	openstack server create \
+		--flavor $FLAVOR_NAME
+		--image $IMAGE_NAME \
+		--network $NETWORK_NAME \
+		--security-group $SEC_GROUP_NAME \
+		--key-name $KEY_NAME \
+		--wait \
+		$SERVER_NAME
+	```
+		
+	Each variable represents a piece of information we have to look 
+	for or, in the cases of `KEY_NAME` and `SERVER_NAME`, arbitrarily 
+	define.
+	
+	Let us begin with the _flavors_ (`FLAVOR_NAME`), which describe 
+	combinations of CPU core count, memory size, and storage capacity. Each 
+	server has a distinct flavor, and to see all available flavors type:
+	
+	```bash
+	openstack flavor list
+	```
+		
+	You will get a pretty long list of flavors. For our 
+	demonstration, we suggest you go with `1C-1GB-20GB`. A server with this 
+	particular flavor will have 1 CPU core, one 
+	[gibibyte](https://en.wikipedia.org/wiki/Gigabyte#Base_2_(binary)) of 
+	RAM, and one disk volume with a capacity of twenty gibibytes. Go ahead 
+	and set `FLAVOR_NAME` accordingly:
+		
+	```bash
+	FLAVOR_NAME="1C-1GB-20GB"
+	```
+		
+	Your server should have an image to boot off of (`IMAGE_NAME`). 
+	For a list of all available images in {extra.brand}} Cloud, type:
+		
+	```bash
+	openstack image list
+	```
+		
+	This time you get a shorter list, but you can still filter for 
+	images with the OS you prefer. For example, filter for Ubuntu:
+		
+	```bash
+	openstack image list | grep Ubuntu
+	```
+		
+	Continue with the `Ubuntu 22.04 Jammy Jellyfish 20220810` image:
+		
+	```bash
+	IMAGE_NAME="Ubuntu 22.04 Jammy Jellyfish 20220810"
+	```
+	
+	You need at least one network in the region you're about to 
+	create your new server (`NETWORK_NAME`). To get the names of all 
+	available networks, type:
+	
+	```bash
+	openstack network list -c Name
+	```
+		
+	```plain
+	+----------------+
+	| Name           |
+	+----------------+
+	| ext-net        |
+	| nordostbahnhof |
+	+----------------+
+	```
+		
+	The `ext-net` is an external network (type `openstack network 
+	show ext-net` for more information). You want to connect your server to 
+	an internal network, so set the `NETWORK_NAME` variable accordingly:
```

Comment:
Sometimes I want to have sort of a discussion with the reader (most of the time these have didactical value, I believe). But since this how-to is somewhat long, let's go with your particular approach (shorter and to the point).